app/models/sample.rb

```diff
@@ -328,6 +337,62 @@ def initiate_s3_cp
     LogUtil.log_err_and_airbrake("Failed to upload S3 sample '#{name}' (#{id}): #{e}")
   end
 
+  def initiate_basespace_file_transfer
```

Comment:
`initiate_basespace_file_transfer` is somewhat misleading as a name because it also completes the transfer... I think most readers would expect it to be named `transfer_basespace_files`